apps/dataset/swagger_api/document_api.py

```diff
@@ -37,3 +37,17 @@ def get_request_body_api():
                                            description="1|2|3 1:向量化|2:生成问题|3:同步文档")
                 }
             )
+
+    class BatchCancel(ApiMixin):
+        @staticmethod
+        def get_request_body_api():
+            return openapi.Schema(
+                type=openapi.TYPE_OBJECT,
+                properties={
+                    'id_list': openapi.Schema(type=openapi.TYPE_ARRAY, items=openapi.Schema(type=openapi.TYPE_STRING),
+                                              title="文档id列表",
+                                              description="文档id列表"),
+                    'type': openapi.Schema(type=openapi.TYPE_INTEGER, title="任务类型",
+                                           description="1|2|3 1:向量化|2:生成问题|3:同步文档", default=1)
+                }
+            )
```

Comment:
The provided code is mostly correct and follows standard API schema definitions using OpenAPI 3 format. However, there are a couple of minor improvements that can be made:

1. **Consistent Schema Formatting**: Ensure consistent formatting throughout the `get_request_body_api` method within `BatchCancel`. The indentation between keys and values in properties should match.

2. **Default Value Clarification**: While not strictly necessary, clearly specifying default values can improve usability and documentation.

Here's the slightly refined version of your code with these considerations addressed:

```python
class BatchCancel(ApiMixin):
    @staticmethod
    def get_request_body_api():
        return openapi.Schema(
            type=openapi.TYPE_OBJECT,
            properties={
                'id_list': openapi.Schema(
                    type=openapi.TYPE_ARRAY,
                    items=openapi.Schema(type=openapi.TYPE_STRING, description="文档ID列表"),
                    title="文档ID列表"
                ),
                'type': openapi.Schema(
                    type=openapi.TYPE_INTEGER,
                    title="任务类型",
                    description="1|2|3: 向量化 | 2: 生成问题 | 3: 同步文档",
                    default=1
                )
            }
        )
```

These changes make it easier to read through and understand the structure of the request body for the `BatchCancel` endpoint.